Solutions/CiscoSEG/Analytic Rules/CiscoSEGPotentialLinkToMalwareDownload.yaml

```diff
@@ -0,0 +1,32 @@
+id: 2e5158e1-9fc2-40ff-a909-c701a13a0405
+name: Cisco SEG - Potential phishing link
+description: |
+  'Detects mails with suspicious links.'

```

Comment:
Can we change the description to "Detects mails with Potential phishing link" as the next query also has the same description.

Reply:
fixed